content/en/building/guides/performance/telemetry.md

```diff
@@ -200,11 +206,305 @@ node get_users_meta_docs.js --mode batch --type telemetry https://admin:pass@loc
+        "contact:ca12_central:create": "11-deb8ed9af0084b7320565dbf91ba0920",
+        "contact:ca12_central:edit": "11-9b974247705d31e05a273b36796e5d9e",
+        "contact:cb20_region:create": "11-bc2f2e729938c120c4d30bc31f7fba28",
+        "contact:cb20_region:edit": "11-5d2f4bb626420a00c6779ee7440101c9",
+        "contact:cb22_region:create": "11-0c733613618f2c937dc877463c387358",
+        "contact:cb22_region:edit": "11-e544ff2386627c3992b98251276e22d5",
+        "contact:cc30_district:create": "11-153e78ab566f066057255220bc4bc4e0",
+        "contact:cc30_district:edit": "11-24355edd2d8414f4d2b9389ada6dd38f",
+        "contact:cc32_district:create": "9-0497fbef8565dbf0d8fb97585303d4cd",
+        "contact:cc32_district:edit": "11-e758c62a1b9505c0b8df3941b2d5bddb",
+        "contact:cd40_commun:create": "11-26cc7683af531080ee02606f7017add6",
+        "contact:cd40_commun:edit": "11-569f8bba69371d044584dbdea32851fc",
+        "contact:cd42_commun:create": "9-01c61755d1b40c8be08bfeedfb8820f9",
+        "contact:cd42_commun:edit": "9-7f6e12e8f28b0944e58be204df2b416d",
+        "contact:ce50_followup:create": "11-7cbbf182aa87df82ba8c77d4bd9650dd",
+        "contact:ce50_followup:edit": "11-95ec94462649c71562e546da9c0702d8",
+        "contact:ce52_followup:create": "11-adcb83d8278f191fb80180d0e61dd9e4",
+        "contact:ce52_followup:edit": "11-60a202887d8528b1bd79a4204070887b",
+        "contact:cf60_sanitary:create": "11-401a5f96a02a8db456f98a5df45a2199",
+        "contact:cf60_sanitary:edit": "11-876b85561d192197021e66f24d0e3869",
+        "contact:cf62_sanitary:create": "11-9b0cf1a9250ebc39a0b39be24bb3bb17",
+        "contact:cf62_sanitary:edit": "11-14343ec14f2c2623ab46512e2a10fc4b",
+        "contact:cg70_supervision:create": "11-7663404abd7a43fc28cbf3e12609ae44",
+        "contact:cg70_supervision:edit": "11-53274a1cde318d03813cb915afdefeb3",
+        "contact:c72_supervision:create": "9-45bf71d3999865a87190ccaf7d5e13d1",
+        "contact:cg72_supervision:edit": "11-45dea06d9dae81ed6d8f9d3ab90ce276",
+        "contact:ch80_village:create": "11-ebfdad62acd97b9f1cbcd2c3d875e0a9",
+        "contact:ch80_village:edit": "11-2da76bd17522fecbfce8960a3c92730f",
+        "contact:ch82_village:create": "11-befb0bf6c09bce5bb7ce1a9ca9347097",
+        "contact:ch82_village:edit": "11-9f665fca707c7edde42a72ca9c2f30b3",
+        "contact:ci90_chw_site:create": "11-2bca19b1e0c7b7ed0bd329c7ae1d9aae",
+        "contact:ci90_chw_site:edit": "11-5b4e596b22bf34e43cd2a5aff56c3539",
+        "contact:ci92_chw_site:create": "11-c051ee845ed6e0a9961fff6e2f8b514e",
+        "contact:ci92_chw_site:edit": "9-00b60f694123dfdfec245ec694d6ef0e",
+        "contact:cj100_household:create": "11-ea9babcf7f0d5ad017e5865ccdadd01d",
+        "contact:cj100_household:edit": "11-bad5365ffce4d5664777ead6dfd8429b",
+        "contact:cj102_household:create": "13-dd6b25dfdd3ecafedbc1893c8e1a4eda",
+        "contact:cj102_household:edit": "11-0ab6d23c170d31d7bdf16a41ad1a3481",
```

Comment:
this is the actual list of forms from the deployment that you sampled this from. I think it would be better to edit this to include default config forms.

Reply:
ah - good thinking! Fixed in 5e56672c